test/sql/types/nested/map/map_from_entries/empty.test

```diff
@@ -0,0 +1,10 @@
+# name: test/sql/types/nested/map/map_from_entries/empty.test
```

Comment:
You're right, but I kind of prefer having many small tests over one big test, for debugging purposes
Though that might be because I don't know how to only test a specific portion of a test file